hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/util/SigReqAdapterUtils.java

```diff
@@ -82,4 +83,12 @@ public PlatformTxnAccessor platformTxn() {
                 };
         return dummyScenario.tokenStore();
     }
+
+    public static TransactionBody txnFrom(final TxnHandlingScenario scenario) {
+        try {
+            return scenario.platformTxn().getTxn();
```

Comment:
null check of scenario is missing

Reply:
I'm not too worried about this since this is a test that will fail on a null scenario